src/users/collection/getItemsInFolderForUser.test.js

```diff
@@ -45,4 +50,18 @@ describe('Users - Collection - getItemsInFolderForUser', () => {
     sort.should.be.equal(by)
     sort_order.should.be.equal(order)
   })
+  it('should accept pagination', async () => {
+    const page = 1
+    const perPage = 3
+    const data = await client.getItemsInFolderForUser({
```

Comment:
This won't work as you are passing an object to the function. You should have wrote `const data = await client.getItemsInFolderForUser(username, folderId, { page, perPage })`

Reply:
Done